pex/variables.py

```diff
@@ -44,6 +45,17 @@ def delete(self, variable):
   def set(self, variable, value):
     self._environ[variable] = str(value)
 
+  def _from_rc(self, rc):
+    ret_vars = {}
+    for filename in [rc, '.pexrc']:
```

Comment:
I guess it already does overlay the files... so maybe that makes sense. The `.pexrc` will always override the `~/.pexrc` file's variables. Is that the idea/intent?